### PR TITLE
Add isSortable flag to determne which column is sortable

### PR DESCRIPTION
**Pull request by jpudysz**
By default it will be always sortable, but if we want to disable sorting e.g on actions column from now it will be possible.

### CODE REVIEW

File: src/head/Column.js [hunk 1]
```diff
@@ -11,25 +9,30 @@ export function Column({column, className, Element}) {
...
-            />
+            {renderSorter(column)}
         </Element>
     );
 }
 
+function renderSorter(column) {
+    return column.isSortable ? (
+        <Sorter
+            onClick={() => onColumnClick(column)}
```
Comment (kazagkazag) [inline src/head/Column.js]: Because this is attached to the sorter, "onSorterClick" would be better name

File: src/head/Column.spec.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+import Sorter from "./Sorter";
+
+describe("Column", () => {
+    const options = {
+        context: {
+            semantic: true
+        }
+    };
+
+    it("should render itself with basic matkup", () => {
```
Comment (kazagkazag) [inline src/head/Column.spec.js]: markup not matkup

File: src/head/Columns.js [hunk 3]
```diff
@@ -25,19 +23,20 @@ function getColumns(columnsDefinitions) {
...
-                    key={sid.generate()}
-                    column={column}
-                />
-            );
-
-        }) : null;
+        const newColumnDefinition = Object.assign({}, column, {
+            sorterComponent,
+            onSort,
+            isSortable: column.isSortable === undefined ? true : column.isSortable
```
Comment (kazagkazag) [inline src/head/Columns.js]: I think that Column component, not Columns, should determine, if it is sortable or not. Parent should not care about props computation of children. It is not responsibility of the Columns component.

File: src/head/Column.js [hunk 4]
```diff
@@ -11,25 +9,30 @@ export function Column({column, className, Element}) {
...
+            sorterComponent={column.sorterComponent}
+        />
+    ) : null;
+}
+
+function onColumnClick(column) {
+    if (column.onSort) {
+        column.onSort(column);
+    }
+}
```
Comment (kazagkazag) [inline src/head/Column.js]: You have changed default behavior. Now onClick is defined on the sorter, not on the entire header cell. Is this intended? You have to click exactly on the sorter to sort column.

**Review by kazagkazag — CHANGES_REQUESTED** [review]
Please do not remove blank line between imports. I like to indicate modules import, component imports and HOC imports be putting blank line between.

Futhermore, specify test for sorting in the header component tests. Header is our 'public' API of this part of the Table. We should not define test for each little piece of the component, because we are interested in working Header as a whole. Thanks for that we can minimize number of tests but keep entire public API of the component tested well.

**Comment by jpudysz** [comment]
Done

File: src/head/Head.test.js [hunk 5]
```diff
@@ -94,29 +96,33 @@ describe("Head", () => {
         expect(columnsElements.at(1).find(".custom-sorter").text()).to.equal("DESC");
     });
 
-    it("should call onSort callback with clicked column", () => {
```
Comment (kazagkazag) [inline src/head/Head.test.js]: Why did you delete this test?

**Review by kazagkazag — CHANGES_REQUESTED** [review]
Great work, only this missing test is a problem.

**Review by kazagkazag — APPROVED** [review]

**Pull request merged**